### PR TITLE
Switch the export controller to stream results directly from PostgreSQL to the response buffer

### LINKED ISSUE #29: New export range end date is exclusive
To download a single day's data, I cannot select the same date for start and end.

In fact, no data from the exported set is of the 'end date'.

This seems unintuitive.

Minor UX issue, recording in case it's easy to fix in the context of something else.

### PR DESCRIPTION
**Pull request by rtyler**
I'm still not entirely sure if the PostgreSQL connection is being suitably
returned to sequelize's connection pool, but I think this is an infrequent
enough operation that it's worth deploying and monitoring of Sentry

Fixes #29, #33

### CODE REVIEW

**Comment by rtyler** [comment]
Might as well just ship this.

File: src/controllers/export.ts [hunk 1]
```diff
@@ -5,27 +5,32 @@
...
-            endDate: req.body.endDate,
-          },
-          user: (req as any).user,
-        })
-        .then((result) => {
-          res.setHeader('Content-Disposition', `attachment; filename=${req.body.type}-${req.body.startDate}.json`);
-          res.setHeader('Content-Type', 'application/json');
-          res.send(result);
+      db.sequelize.connectionManager.getConnection().then((pgConn) => {
+        const query = new QueryStream('SELECT * FROM events WHERE type = $1 AND "createdAt" > $2 AND "createdAt" <= $3', [
```
Comment (daniel-beck) [inline src/controllers/export.ts]: Unclear semantics with dates. If they're interpreted as midnight, this does not actually address #29. If this was tested, great.

Comment (daniel-beck) [inline src/controllers/export.ts]: Is SQL injection possible?

Comment (daniel-beck) [inline src/controllers/export.ts]: This looks like end date is not mandatory, when before, IIRC, it was not. Fine if intentional, just noting.

**Pull request merged**